### PR TITLE
Recommendation System 

### PR DESCRIPTION
**Pull request by RobertBeaudenon**
Displayed Recommendations on the dashboard of each organization:
1) Frequently audited Bins (top 5)
2) Frequently audited Parts (top5)
3) Frequently lost Items (top5)

### CODE REVIEW

File: server/audit/views.py [hunk 1]
```diff
@@ -383,3 +385,36 @@ def check_item(self, request):
                 status=status.HTTP_400_BAD_REQUEST)
 
         return response
+
+
+class RecommendationViewSet(LoggingViewset):
+    """
+    Allows obtaining all clients and updating them
```
Comment (makchamp) [inline server/audit/views.py:392]: Can you update this comment ?

File: server/audit/views.py [hunk 2]
```diff
@@ -383,3 +385,36 @@ def check_item(self, request):
...
+    """
+    http_method_names = ['get']
+    serializer_class = GetBinToSKSerializer
```
Comment (makchamp) [inline server/audit/views.py:395]: Is this the correct serializer for this viewset ? I see you added RecommendationBinSerializer but havent used it anywhere.

File: server/audit/serializers.py [hunk 3]
```diff
@@ -31,6 +29,15 @@ class Meta:
...
 
+class RecommendationBinSerializer(serializers.ModelSerializer):
+    customuser = UserAuditSerializer(read_only=True, many=False)
+    init_audit = AuditSerializer(read_only=True, many=False)
+
+    class Meta:
+        model = BinToSK
+        fields = ['bin_id','Bin','init_audit', 'customuser']
+
+
```
Comment (makchamp) [inline server/audit/serializers.py:40]: This serializer isnt used anywhere

**Review by makchamp — COMMENTED** [review]
Thanks this will provide a good base to work off on for the recommendation system. I just made a couple small comments to address.

File: server/audit/views.py (hunk 1 above)
Comment (RobertBeaudenon) [reply server/audit/views.py:392]: Yes definitely

File: server/audit/views.py (hunk 2 above)
Comment (RobertBeaudenon) [reply server/audit/views.py:395]: Yes this is the one used I'm going to delete the other serializer

File: server/audit/serializers.py (hunk 3 above)
Comment (RobertBeaudenon) [reply server/audit/serializers.py:40]: I'm going to delete it

**Review by makchamp — APPROVED** [review]
Ok thanks for making the changes.

**Pull request merged**